src/features/import-maps.js

```diff
@@ -39,6 +39,9 @@ function processScripts () {
       script.sp = true;
       var fetchPromise = script.src ? fetch(script.src, { integrity: script.integrity }).then(function (res) {
         return res.text();
+      }).catch((err) => {
+        console.error((process.env.SYSTEM_PRODUCTION ? errMsg(7) : errMsg(7, "Failed to fetch module - wrong HTTP status")) + "\n" + script.src);
```

Comment:
The production error messages strip the text to instead point to the URL. Also perhaps we should still log an error instance?

```suggestion
        console.error(Error(errMsg(7, process.env.SYSTEM_PRODUCTION ? script.src : 'Failed to fetch ' + script.src)));
```

Reply:
Yes keeping the Error sounds like a good idea.